install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

```diff
@@ -0,0 +1,154 @@
+# Migration Guide from Thanos to Kubecost 2.0 (Aggregator)
+
+This tutorial is intended to help our users migrate from the legacy Thanos federation architecture to [Kubecost 2.0 (Aggregator)](aggregator.md). There are a few requirements in order to successfully migrate to Kubecost 2.0. This new version of Kubecost includes a new backend Aggregator which handles the ETL data built from source metrics more efficiently. Kubecost 2.0 will provide new features, optimize the Kubecost's UI performance, and enhance the user experience.
+
+## Key changes
+* Assets and Allocations are now paginated
+   * Frontend/API users never have full set of information
+   * Use `offset`/`limit` parameters to govern pagination
+* New data available for querying every 2 hours (can be adjusted)
+* Embedded DuckDB database serves queries
+   * Data no longer queried directly from bingen files
+   * Substantial query speed improvements even when pagination not in effect
+* Data ingested into independent Aggregator component
+* Idle (sharing), Cluster Management sharing, and Network are computed a priori
+* Distributed tracing integrated into core workflows
+* No more pre-computed "agg stores"
+   * Request-level caching still in effect
+
+<details>
+
+<summary>Diagram of Aggregator:</summary>
+
+![aggregator-diagram](/images/aggregator/aggregator-diagram.png)
+
+</details>
+
+## Migration Path 
+
+<details>
+
+<summary>Diagram of Aggregator:</summary>
+
+![migration-diagram](/images/aggregator/migration-diagram.png)
+
+</details>
+
+* All steps are done on the primary except for step 8.
+* Nothing needs to be done on the secondary clusters for the migration to be successful. The Thanos sidecar on the secondary clusters will have no impact on the migration or functionality.
+* Once Aggregator is enabled, all queries hit the Aggregator container and NOT cost-model via the reverse proxy.
+* ETL Utils does not destroy the Thanos data, it creates additional directories in the object store.
+* For larger environments, the StorageClass must have 1GBPS throughput.
+* Having enough storage is important and will vary based on environment.
+
+## Prerequisites
+
+To migrate from Thanos multi-cluster federated architecture to Aggregator, users *must* complete the following steps:
+
+### Step 1: Use the existing Thanos object store or create a new dedicated object store
```

Comment:
Is this even a step? We need to show users how to create a new object store if they prefer that option.

Reply:
I think the right way to clarify this is that users of metric federation tools other than Thanos which don't require an object store (e.g. AMP) will not have a bucket yet and thus will need to create one.

Reply:
I just addressed this by adding some references to our long-term-storage docs. So users who were on tools like AMP have instructions to set up a bucket.